src/engraving/dom/segment.cpp

```diff
@@ -1936,7 +1960,13 @@ bool Segment::notChordRestType() const
         || segmentType() == SegmentType::EndBarLine
         || segmentType() == SegmentType::BarLine
         || segmentType() == SegmentType::KeySigAnnounce
-        || segmentType() == SegmentType::TimeSigAnnounce) {
+        || segmentType() == SegmentType::TimeSigAnnounce
+        || segmentType() == SegmentType::ClefRepeatAnnounce
+        || segmentType() == SegmentType::TimeSigRepeatAnnounce
+        || segmentType() == SegmentType::KeySigRepeatAnnounce
+        || segmentType() == SegmentType::ClefStartRepeatAnnounce
+        || segmentType() == SegmentType::TimeSigStartRepeatAnnounce
+        || segmentType() == SegmentType::KeySigStartRepeatAnnounce) {
```

Comment:
Wait... why doesn't this function simply `return segmentType() != SegmentType::ChordRest`? 😅